### PR TITLE
bugfix/14952-organize-tech-indicators-docs

### LINKED ISSUE #14952: Organize Tech. Indicators in docs:
https://www.highcharts.com/docs/stock/technical-indicator-series

The list is too long, perhaps put in the table?

### PR DESCRIPTION
**Pull request by karolkolodziej**
Fixed #14952, divided indicators into groups in the documentation.

Changes based on https://www.investopedia.com/terms/t/technicalindicator.asp

### CODE REVIEW

**Review by pawellysy — APPROVED** [review]
Looks Good!

File: docs/stock/technical-indicator-series.md [hunk 1]
```diff
@@ -3,91 +3,36 @@ Technical indicators
...
+| ------------- | ------------- |
+| [Acceleration Bands](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/stock/indicators/acceleration-bands/)| [Absolute Price Oscillator](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/stock/indicators/apo/)  |
+|  [Bollinger Bands](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/stock/indicators/bollinger-bands/)   | [A/D (Accumulation/Distribution)](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/stock/indicators/accumulation-distribution/)   |
+| [CMF (Chaikin Money Flow)](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/stock/indicators/cmf/)   | [Aroon](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/stock/indicators/aroon/)   |
+| [Double EMA (Exponential Moving Average)](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/stock/indicators/dema/)   | [Aroon Oscillator](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/stock/indicators/aroon-oscillator/)   |
```
Comment (raf18seb) [inline docs/stock/technical-indicator-series.md]: The official full name for DEMA is `(Double Exponential Moving Average)`. The same is for TRIX and also in your other pull requests

**Review by raf18seb — CHANGES_REQUESTED** [review]

File: docs/stock/technical-indicator-series.md [hunk 2]
```diff
@@ -3,91 +3,37 @@ Technical indicators
...
+
+A full list of supported technical indicators could be divided into two main groups.  Overlays use the same scale and are plotted on the same xAxis and yAxis as the main series. The oscillator requires additional yAxis because they oscillate between different extremes.
+
+
+| Overlays  |Oscillators |
+| ------------- | ------------- |
+| [Acceleration Bands](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/stock/indicators/acceleration-bands/) | [Absolute Price Oscillator](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/stock/indicators/apo/) |
+| [Bollinger Bands](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/stock/indicators/bollinger-bands/) | [A/D (Accumulation/Distribution)](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/stock/indicators/accumulation-distribution/) |
+| [Double EMA (Exponential Moving Average)](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/stock/indicators/dema/) | [Aroon](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/stock/indicators/aroon/) |
```
Comment (raf18seb) [inline docs/stock/technical-indicator-series.md:15]: The official full name for DEMA is `Double Exponential Moving Average`. The same for TRIX - `Tripple Exp...`.
Please, correct the same in your other pull requests.

**Review by raf18seb — CHANGES_REQUESTED** [review]

File: docs/stock/technical-indicator-series.md (hunk 2 above)
Comment (karolkolodziej) [reply docs/stock/technical-indicator-series.md:15]: I think you meant TEMA instead of TRIX?

Comment (raf18seb) [reply docs/stock/technical-indicator-series.md]: Yes, sorry my bad! TEMA is `Triple Exponential Moving Average`, TRIX is `Triple Exponential Average`. However, when doing small research now, I found different websites that conflict with each other. Some of them call TRIX `Triple Exponential Average`, but the rest call it `Triple Exponential Moving Average`. Please, make sure which one is correct (I assume the naming in my second sentence is correct).

Comment (karolkolodziej) [reply docs/stock/technical-indicator-series.md]: Our code says that `TRIX` stands for `Triple exponential average`, I think we should stick with that.
https://github.com/highcharts/highcharts/blob/cc1adf69558e1aea9b731c27e7c46d0a17911981/ts/Stock/Indicators/TRIX/TRIXIndicator.ts#L42

**Review by raf18seb — APPROVED** [review]
Thanks!

**Review by TorsteinHonsi — APPROVED** [review]

**Pull request merged**